deps/GraphBLAS/Source/GxB_Matrix_tuple_iter.c

```diff
@@ -70,15 +86,18 @@ GrB_Info GxB_MatrixTupleIter_iterate_range
 	GrB_Index startRowIdx,      // row index to start with
 	GrB_Index endRowIdx         // row index to finish with
 ) {
-	GB_WHERE("GxB_MatrixTupleIter_iterate_range (iter, startRowIdx, endRowIdx)");
-	GB_RETURN_IF_NULL(iter);
+	GB_WHERE("GxB_MatrixTupleIter_iterate_range (iter, startRowIdx, endRowIdx)") ;
+	GB_RETURN_IF_NULL(iter) ;
+
+	// Return an empty iterator, should caller ignore returned error.
+	_EmptyIterator(iter) ;
```

Comment:
I personally feel like callers just shouldn't ignore errors, rather than having this logic.

That said, this does work fine, and I'm not opposed to keeping it unless we want to share our tuple iter logic with other projects (in that case, we should strive for something cleaner here).

Reply:
I don't see a problem here, the caller gave us a pointer to modify, in case of an error caller shouldn't make any assumption on the internal state of its iterator.